filter_dom.py

```
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
```

Ruff (0.8.2)

Shebang is present but file is not executable

(EXE001)